modules/test/playwright/pages/commerce/commerce-organization-web/organizationManagementPage.ts

```diff
@@ -82,13 +88,22 @@ export class OrganizationManagementPage {
 				.locator('g.chart-item-organization')
 				.filter({hasText: organizationName});
 		};
+		this.searchInput = page.getByPlaceholder('Type Here');
 		this.userNode = (userName) => {
 			return this.chart
 				.locator('g.chart-item-user')
 				.filter({hasText: userName});
 		};
 	}
 
+	async searchedEntry(name: string) {
+		return this.page.getByRole('menuitem', {name: new RegExp(name)});
+	}
+
+	async verifyTextIsShown(name: string) {
+		return this.page.getByText(name);
+	}
+
```

Comment:
because these return a locator why a dedicated method and not the usual dynamic locator? (like the `userNode`)